blazor/datagrid/cell.md

```diff
@@ -129,222 +243,377 @@ The appearance of cells can be customized by using the [QueryCellInfo](https://h
+The Syncfusion Grid allows you to display information about the grid columns to the user when they hover over them with the mouse.
 
-There are three types of [ClipMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ClipMode). They are:
+### Display custom tooltip for columns
 
-* **Clip**: Truncates the cell content when it overflows its area.
-* **Ellipsis**: Displays ellipsis when the cell content overflows its area.
-* **EllipsisWithTooltip**: Displays ellipsis when the cell content overflows its area, also it will display the tooltip while hover on ellipsis is applied.
+The Grid provides a feature to display custom tooltips for its columns using the [SfTooltip](https://blazor.syncfusion.com/documentation/tooltip/getting-started) component. This allows you to provide additional information about the columns when the user hovers over them.
 
-N> By default, [Columns.ClipMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ClipMode) value is **Ellipsis**.
+To enable custom tooltips for columns in the Grid,you can use the [Column Template](https://blazor.syncfusion.com/documentation/datagrid/column-template) feature by rendering the components inside the template
+
+This is demonstrated in the following sample code, where the tooltip for the **FirstName** column is rendered using `Column Template`.
 
 ```cshtml
 @using Syncfusion.Blazor.Grids
+@using Syncfusion.Blazor.Popups
 
-<SfGrid DataSource="@Orders" AllowPaging="true" Height="315">
+<SfGrid DataSource="@Employees">
     <GridColumns>
-        <GridColumn Field=@nameof(Order.RollNo) HeaderText="Roll No" Width="140"></GridColumn>
-        <GridColumn Field=@nameof(Order.Name) HeaderText="Name of the inventor" ClipMode="ClipMode.Clip" Width="70"></GridColumn>
-        <GridColumn Field=@nameof(Order.PatentFamilies) HeaderText="No of patent families" ClipMode="ClipMode.Ellipsis" Width="80"></GridColumn>
-        <GridColumn Field=@nameof(Order.Country) HeaderText="Country" Width="100"></GridColumn>
-        <GridColumn Field=@nameof(Order.MainFields) HeaderText="Main fields of Invention" ClipMode="ClipMode.EllipsisWithTooltip" Width="120"></GridColumn>
+        <GridColumn Field=@nameof(EmployeeData.EmployeeID) HeaderText="Employee ID" TextAlign="TextAlign.Right" Width="120"></GridColumn>
+        <GridColumn Field=@nameof(EmployeeData.FirstName) HeaderText="First Name" Width="130">
+            <Template>
+                @{
+                    var employee = (context as EmployeeData);
+                    Count++;
+                    <SfTooltip @key="@Count" Position="Position.BottomLeft">
+                        <TooltipTemplates>
```

Comment:
change the code example to latest version. TooltipTemplates were marked as absolete

Reply:
modified